core/modules/qproc/SConscript

```diff
@@ -6,38 +6,42 @@ import os
 
 standardModule(env, test_libs='log4cxx')
 
+
 # build kvmap.h
 # If you need to rebuild the map, use:
 # ./admin/bin/qserv-admin.py < admin/examples/testMap_generateMap
 def charArrayGenerator(filename):
     bSize = 70
-    with open(filename,"rb") as f:
+    with open(filename, "r") as f:
```

Comment:
I'm not sure this is the right thing to do, I think the intent here is to make the stream of bytes rather than chars, it gets translated into C++ char[]. And it probably does not matter in practice as we don't have Unicode in input file.